csrc/cache_kernels.cu

```diff
@@ -143,12 +144,12 @@ void copy_blocks(
 
 namespace vllm {
 
-template<typename scalar_t>
+template<typename scalar_t, typename cache_t, bool is_fp8_e5m2_kv_cache>
```

Comment:
is_fp8_e5m2_kv_cache ，In the cuda kernel file, it is recommended that this parameter can be designed to be more general, such as is_quant_kv_cache
, inside the function, determine whether it is fp8_e5m2 based on is_quant_kv_cache, which facilitates subsequent non-fp8 cuda-kernel access.

Reply:
I think it is good to keep clean and simple inside cuda kernel. Too much if-else inside cuda kernel may have side-effect on  performance.